go/runtime/history/db.go

```diff
@@ -28,10 +28,10 @@ var (
 	//
 	// Value is CBOR-serialized roothash.AnnotatedBlock.
 	blockKeyFmt = keyFormat.New(0x02, uint64(0))
-	// roundResultsKeyFmt is the round result index key format.
-	//
-	// Value is CBOR-serialized roothash.RoundResults.
-	roundResultsKeyFmt = keyFormat.New(0x03, uint64(0))
+
+	// deprecatedRoundResultsKeyFmt is deprecated. It may be removed once
+	// clean-up is done, since history may still have legacy data stored.
```

Comment:
```suggestion
	// deprecatedRoundResultsKeyFmt is the deprecated round result index key format.
	//
	// Value is CBOR-serialized roothash.RoundResults.
	//
	// Deprecated: This key format is deprecated and will be removed in future versions
	// after we clean up the remaining round results from the block history (#6098). 
```
Whoever will be reading this will have no idea what the cleanup is neither about which history you are talking about. Be more general or more exact.

```suggestion
	// deprecatedRoundResultsKeyFmt is the deprecated round result index key format.
	//
	// Value is CBOR-serialized roothash.RoundResults.
	//
	// Deprecated: This key format is deprecated and will be removed in future versions.
```